code/game/machinery/fabricators/modular_fabricator.dm

```diff
@@ -498,12 +498,12 @@
 	else
 		for(var/i in 1 to multiplier)
 			var/obj/item/new_item = new being_built.build_path(A)
-			new_item.materials.Cut()	//appearantly the material datum gets initialized in a subsystem so there is no need to qdelete it but we still need to empty the list
-			for(var/mat in materials_used)
-				new_item.materials[mat] = materials_used[mat] / multiplier
 
 			if(length(picked_materials))
 				new_item.set_custom_materials(picked_materials, 1 / multiplier) //Ensure we get the non multiplied amount
+	var/location = get_step(src,(dir))
```

Comment:
We do have custom lathe code, so I was a bit worried if this hacky bit I wrote would work. But Testing evidence seems to show it did. 

If you can think of a cleaner way to do this, let me know

Reply:
No longer an issue, as Material properties are an inclusive flag, rather than an exclusive one